### PR TITLE
Typing for useragent [Types-2.0]

### PR DESCRIPTION
**Pull request by geoffreak**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

---

Typings for [`useragent`](https://github.com/3rd-Eden/useragent)

### CODE REVIEW

**Comment by geoffreak** [comment]
Looks like the tests failed because of an unrelated issue with `ejs`.

**Comment by andy-ms** [comment]
See  #11972

**Comment by geoffreak** [comment]
Any chance this can get pulled now?

File: useragent/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": false,
```
Comment (andy-ms) [inline useragent/tsconfig.json]: Please make this `true`

Comment (geoffreak) [inline useragent/tsconfig.json]: No problem. Is this something all PRs should have from now on?

Comment (andy-ms) [inline useragent/tsconfig.json]: Yep. Just updated the README to say so too. Old packages should also be gradually updated to support this.

Comment (geoffreak) [inline useragent/tsconfig.json]: Changed and pushed on both my open PRs

**Pull request merged**